administration/src/project-configs/helper/index.ts

```diff
@@ -0,0 +1,7 @@
+import { ProjectConfig } from '../getProjectConfig'
+
+export const getCsvHeaders = (projectConfig: ProjectConfig): string[] => [
```

Comment:
Do you think this is good naming? Or should we change this to `columnNames` or similar?

Reply:
hm for now i think its okay because it serves this particular value. But if we combine name, expiry date and extension into one array to make it more flexible and to enable better reordering of form fields we might not need to explicitly define here the columns names in the array but just loop through and then we could rename it to `getColumnNames`